lib/whitehall_importer/integrity_checker.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module WhitehallImporter
+  class IntegrityChecker
+    attr_reader :edition
+
+    def initialize(edition)
+      @edition = edition
+    end
+
+    def valid?
+      problems.empty?
+    end
+
+    def problems
+      problems = []
+      problems += content_problems
+      problems += image_problems
+      problems += organisation_problems
+
+      problems
```

Comment:
What about 

```ruby
[].concat(content_problems, image_problems, organisation_problems)
```

Reply:
I didn't realise `concat` is a lot faster.

Reply:
I could just do `content_problems + image_problems + organisation_problems`

Reply:
Even better